ghost/core/core/server/data/schema/default-settings/default-settings.json

```diff
@@ -1,5 +1,9 @@
 {
     "core": {
+        "last_mentions_report_email_timestamp": {
+            "defaultValue": null,
+            "type": "datetime"
```

Comment:
I don't see any other usage of type datetime, is this supported?

Reply:
Good shout  - we manually parse it when we use it, I guess should change to "string"?